packages/example-codehub/src/codehub-application.ts

```diff
@@ -3,23 +3,32 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Application, CoreBindings} from '@loopback/core';
+import {Application, ApplicationConfig} from '@loopback/core';
+import {RestServer, RestBindings} from '@loopback/rest';
 import {UserController, HealthController} from './controllers';
 
 export class CodeHubApplication extends Application {
-  constructor() {
-    super();
+  constructor(options?: ApplicationConfig) {
+    super(options);
 
     const app = this;
-
-    app.api({
-      swagger: '2.0',
-      info: {
-        title: 'CodeHub',
-        version: require('../package.json').version,
-      },
-      basePath: '/',
-      paths: {},
+    app.server(RestServer);
+    // We can't use the async version in the ctor!
+    const server = app.getServer(RestServer);
+
+    // tslint:disable-next-line:no-floating-promises
+    server.then(s => {
```

Comment:
AFAICT, you don't need anything from the `server` object to build the Swagger API description. Can we remove `app.getServer(RestServer)` call completely?